muselsl/muse.py

```diff
@@ -239,15 +249,20 @@ def _handle_eeg(self, handle, data):
 
             # push data
             self.callback_eeg(self.data, timestamps)
-            self._init_sample()
 
+            # save last timestamp for disconnection timer
+            self.last_timestamp = timestamps[-1]
+
+            # reset sample
+            self._init_sample()
 
     def _init_control(self):
         """Variable to store the current incoming message."""
         self._current_msg = ""
 
     def _subscribe_control(self):
-        self.device.subscribe('273e0001-4c4d-454d-96be-f03bac821358', callback=self._handle_control)
+        self.device.subscribe(
+            '273e0001-4c4d-454d-96be-f03bac821358', callback=self._handle_control)
```

Comment:
I would add the subscribe const to constants as well

Reply:
All constants insterted!